Sources/KIF/Classes/KIFUIViewTestActor.m

```diff
@@ -155,6 +156,26 @@ - (instancetype)usingFirstResponder;
     return [self usingPredicate:predicate];
 }
 
+- (instancetype)usingCustomActionWithName:(NSString *)name
+{
+    NSPredicate *predicate = [NSPredicate predicateWithBlock:^BOOL(id evaluatedObject, NSDictionary *bindings) {
+        NSArray *actions = [evaluatedObject accessibilityCustomActions];
```

Comment:
Every element that we iterate over in the view & accessibility element hierarchy is guaranteed to respond to this selector? Or should we add a `respondsToSelector:` check?

Reply:
This is a property on NSObject defined thusly in UIAccessibility.h
Though a respondsToSelector: check couldn't hurt.

`@property (nullable, nonatomic, strong) NSArray <UIAccessibilityCustomAction *> *accessibilityCustomActions API_AVAILABLE(ios(8.0));`